test/TemplateTests.targets.in

```diff
@@ -37,30 +37,40 @@ Copyright (c) .NET Foundation. All rights reserved.
   <Choose>
     <When Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'" >
 
+      <PropertyGroup>
```

Comment:
This section should be updated to take into account the changes you made in https://github.com/aspnet/websdk/pull/346/commits/9e595f7b78ff0b0bee35ad47d7af417adee8b7b7. The logic in the websdk and templating should be identical.